src/disco/containers/DiscoPane.js

```diff
@@ -20,6 +20,21 @@ class DiscoPane extends React.Component {
     const { results } = this.props;
     return (
       <div id="app-view" ref="container">
+        <header>
+          <div className="content">
+            <h1>{_('Personalize Your Firefox')}</h1>
+            <p>{_(`There are thousands of add-ons that let you make Firefox all your
```

Comment:
I was thinking of stripping the leading indenation but then I'm not sure if extraction will see that. I think it best to cross that bridge when we do the extraction.